java/ql/src/Security/CWE/CWE-287/AndroidInsecureKeys.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @name Insecurely generated keys for local authentication
+ * @description Keys used for local biometric authentication should be generated with secure parameters.
```

Comment:
```suggestion
 * @description Generation of keys with insecure parameters for local biometric authentication can allow attackers with physical access to bypass authentication checks.
```

Small suggestion to re-word this so it follows the "X causes behavior Y" pattern for descriptions, but up to you if this is too long and you'd rather stick with what you have 👍🏼